ecosystem.jenkins.config.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
 				PORT: '9901',
 				SERVICE_BROKER: 'redis://localhost:6379/0',
 				STRICT_READINESS_CHECK: true,
+				ENABLE_WS_RATELIMIT: false,
```

Comment:
Please add the following environment variables as well and please update as a comment if `WS_RATELIMIT_DURATION` is in seconds or millisecs. The same config needs to be updated for `gateway` microservice in `ecosystem.core3.config.js` file as well.
```suggestion
				ENABLE_WS_RATELIMIT: false,
				WS_RATELIMIT_POINTS: 5,
				WS_RATELIMIT_DURATION: 1, // in seconds
```